x-pack/plugins/enterprise_search/server/integrations.ts

```diff
@@ -377,265 +384,282 @@ export const registerEnterpriseSearchIntegrations = (
+          type: 'svg',
+          src: http.basePath.prepend('/plugins/enterpriseSearch/assets/source_icons/oracle.svg'),
+        },
+      ],
+      shipper: 'enterprise_search',
+      isBeta: false,
+    });
 
-  customIntegrations.registerCustomIntegration({
-    id: 'ms_sql',
-    title: i18n.translate('xpack.enterpriseSearch.workplaceSearch.integrations.msSqlName', {
-      defaultMessage: 'Microsoft SQL',
-    }),
-    description: i18n.translate(
-      'xpack.enterpriseSearch.workplaceSearch.integrations.msSqlDescription',
-      {
-        defaultMessage: 'Search over your content on Microsoft SQL Server with Enterprise Search.',
-      }
-    ),
-    categories: ['enterprise_search', 'custom', 'elastic_stack', 'datastore'],
-    uiInternalPath: '/app/enterprise_search/content/search_indices/new_index?method=connector',
-    icons: [
-      {
-        type: 'svg',
-        src: http.basePath.prepend(
-          '/plugins/enterpriseSearch/assets/source_icons/microsoft_sql.svg'
-        ),
-      },
-    ],
-    shipper: 'enterprise_search',
-    isBeta: false,
-  });
+    customIntegrations.registerCustomIntegration({
+      id: 'ms_sql',
+      title: i18n.translate('xpack.enterpriseSearch.workplaceSearch.integrations.msSqlName', {
+        defaultMessage: 'Microsoft SQL',
+      }),
+      description: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.integrations.msSqlDescription',
+        {
+          defaultMessage:
+            'Search over your content on Microsoft SQL Server with Enterprise Search.',
+        }
+      ),
+      categories: ['enterprise_search', 'custom', 'elastic_stack', 'datastore'],
+      uiInternalPath: '/app/enterprise_search/content/search_indices/new_index?method=connector',
+      icons: [
+        {
+          type: 'svg',
+          src: http.basePath.prepend(
+            '/plugins/enterpriseSearch/assets/source_icons/microsoft_sql.svg'
+          ),
+        },
+      ],
+      shipper: 'enterprise_search',
+      isBeta: false,
+    });
 
-  customIntegrations.registerCustomIntegration({
-    id: 'ms_sql',
-    title: i18n.translate('xpack.enterpriseSearch.workplaceSearch.integrations.networkDriveName', {
-      defaultMessage: 'Network Drive',
-    }),
-    description: i18n.translate(
-      'xpack.enterpriseSearch.workplaceSearch.integrations.netowkrDriveDescription',
-      {
-        defaultMessage: 'Search over your Network Drive content with Enterprise Search.',
-      }
-    ),
-    categories: ['enterprise_search', 'custom', 'elastic_stack', 'file_storage'],
-    uiInternalPath: '/app/enterprise_search/content/search_indices/new_index?method=connector',
-    icons: [
-      {
-        type: 'svg',
-        src: http.basePath.prepend(
-          '/plugins/enterpriseSearch/assets/source_icons/network_drive.svg'
-        ),
-      },
-    ],
-    shipper: 'enterprise_search',
-    isBeta: false,
-  });
+    customIntegrations.registerCustomIntegration({
+      id: 'ms_sql',
```

Comment:
Hi! I noticed this ID seems to be wrong for this integration while resolving conflicts for #153216 (seems like it was already wrong before your changes). Then I noticed that Network Drive is registered on [L168](https://github.com/TattdCodeMonkey/kibana/blob/160b6ff7ac13ae2a1a1477bf8aea9400cbc0712c/x-pack/plugins/enterprise_search/server/integrations.ts#L168) too. Should that one be removed and the ID here updated to `network_drive`?

Btw I am happy to make the changes in my PR if you want since I am about to merge that one.

Reply:
Feel free to update in your PR, I agree this should be `network_drive` and that it was already incorrect. My guess is the `ms_sql` item was copied and we forgot to change the id.

Reply:
`network_drive` is also already used for the workplace search integration, so we could name this one `network_drive_connector` to keep it unique.